### ISSUE TITLE
setting bedspawn feature improvement

### ISSUE THREAD

**Issue by Ahmadh26** [Zrips/CMI#3143]
**Description of feature/improvement request:**
Quoting from CMI Config file (lines 387 - 389)
```
  # When set to true by interacting with bed, regular home location will be created
  # When set to false vanilla bed home location will be set
  BedInteraction: true
```

is it possible to make that true only during the day and not nights/storm?

---
### Detailed explanation of what it should do, when and in what situation: ###

Make it so that bed spawn is set if and only if:
It is day time and there is no storm. since players usually sleep during a storm to pass the storm.

### Basic explanation as to why this feature is needed and/or useful to have: ###

basically when players are trying to sleep, their beds get re-set to the location they are sleeping in, it gets annoying when you're far away from your home, just setup a bed to sleep and pass the night then the bed spawn get's updated.

**Comment by Zrips** [comment]
What i can add is option to require shift+right click to set home location which would prevent situations like that. Other than this, we still need option to set home location independent of weather. So next update will have option to request shift click.
